graylog2-server/src/main/java/org/graylog2/contentpacks/facades/UrlWhitelistFacade.java

```diff
@@ -0,0 +1,124 @@
+/**
+ * This file is part of Graylog.
+ *
+ * Graylog is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Graylog is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Graylog.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.graylog2.contentpacks.facades;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.ImmutableSet;
+import org.graylog2.contentpacks.EntityDescriptorIds;
+import org.graylog2.contentpacks.model.ModelId;
+import org.graylog2.contentpacks.model.ModelType;
+import org.graylog2.contentpacks.model.constraints.GraylogVersionConstraint;
+import org.graylog2.contentpacks.model.entities.Entity;
+import org.graylog2.contentpacks.model.entities.EntityDescriptor;
+import org.graylog2.contentpacks.model.entities.EntityExcerpt;
+import org.graylog2.contentpacks.model.entities.EntityV1;
+import org.graylog2.contentpacks.model.entities.NativeEntity;
+import org.graylog2.contentpacks.model.entities.NativeEntityDescriptor;
+import org.graylog2.contentpacks.model.entities.references.ValueReference;
+import org.graylog2.plugin.Version;
+import org.graylog2.system.urlwhitelist.UrlWhitelistService;
+import org.graylog2.system.urlwhitelist.WhitelistEntry;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Inject;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import static org.graylog2.contentpacks.model.ModelTypes.URL_WHITELIST_ENTRY_V1;
+
+public class UrlWhitelistFacade implements EntityFacade<WhitelistEntry> {
+    private static final Logger LOG = LoggerFactory.getLogger(UrlWhitelistFacade.class);
```

Comment:
Logger is not used. Not a big deal through.

Reply:
Right. Removed it.